rasa/core/exporter.py

```diff
@@ -279,44 +276,3 @@ def _get_events_for_conversation_id(
             events_with_conversation_id.append(event)
 
         return events_with_conversation_id
-
-    def _sort_and_select_events_by_timestamp(
-        self, events: Iterable[Dict[Text, Any]]
-    ) -> List[Dict[Text, Any]]:
-        """Sort list of events by ascending timestamp, select events within time range.
-
-        Args:
-            events: List of serialized events to be sorted and selected from.
-
-        Returns:
-            List of serialized and sorted (by timestamp) events within the requested
-            time range.
-
-        Raises:
-             `NoEventsInTimeRangeError` error if no events are found within the
-             requested time range.
-
-        """
-        logger.debug(f"Sorting and selecting from {len(events)} total events found.")
-        # sort the events by timestamp just in case they're not sorted already
-        events = sorted(events, key=lambda x: x["timestamp"])
```

Comment:
Shouldn't we move these lines after L243 in your diff?

Reply:
events don't need to be sorted, we get them from a tracker where we know they are sorted https://github.com/RasaHQ/rasa/pull/11623/files#diff-d02c9371465cb5be594493da79a984200c506a1904675721a8d0a96c397d47d5R235